book/src/voluntary-exit.md

```diff
@@ -0,0 +1,72 @@
+# Voluntary exits
+
+A validator may chose to voluntary stop performing duties (proposing blocks and attesting to blocks) by submitting
+a voluntary exit transaction to the beacon chain.
+
+A validator can initiate a voluntary exit provided that the validator is currently active (has not been slashed) and has been active for at least 256 epochs (~27 hours) since it has been activated.
```

Comment:
```suggestion
A validator can initiate a voluntary exit provided that the validator is currently active, has not been slashed and has been active for at least 256 epochs (~27 hours) since it has been activated.
```